sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/ecert-part-time-process-integration.scheduler.e2e-spec.ts

```diff
@@ -294,5 +452,121 @@ describe(
         DisbursementScheduleStatus.Pending,
       );
     });
+
+    /**
+     * Create and save required number of notifications for student and ministry.
+     * @param notificationsCount number of notifications to create.
+     * @param student related student.
+     * @param disbursementId related disbursement id.
+     */
+    async function saveNotifications(
+      notificationsCount: number,
+      student: Student,
+      disbursementId: number,
+      daysAhead?: number,
+    ): Promise<void> {
+      while (notificationsCount > 0 && notificationsCount--) {
```

Comment:
Please use a seperate constant for the counter and not manipulate the method argument.

Please move the `notificationsCount--` inside the while implementation.